{
  try {
  }
  finally {
    try {
      new Callable<Object>(){
        @Override public Object call() throws Exception {
          return null;
        }
      }
;
    }
 catch (    Exception e) {
    }
  }
}
